src/main/python/KeyboardToNetworkTables.py

```diff
@@ -0,0 +1,69 @@
+from NetworkTableManager import NetworkTableInstance, NetworkTable, NetworkTableClient
+
+import sys
+import time
+from typing import Callable, TypeVar
+
+from pynput import keyboard
+
+SpecialKey = TypeVar('SpecialKey')
+ASCIIKey = TypeVar('ASCIIKey')
+UndefinedKey = TypeVar('UndefinedKey')
+
+KEYBOARD_EVENT_CHECKING_COOLDOWN_SECONDS = 0.01
+KEYBOARD_KEYS_TABLE = "Keyboard/Keys"
+CLIENT_NAME = "KeyboardToNetworkTables"
+IP = sys.argv[1]
+
+
+def key_type_checker(key: object) -> TypeVar:
+    if hasattr(key, "name"):
+        return SpecialKey
+    elif hasattr(key, "char"):
+        return ASCIIKey
+
+    return UndefinedKey
+
+
+def keys_handler(table: NetworkTableInstance, is_pressed: bool) -> Callable:
```

Comment:
what do function do?

Reply:
is `keys_event_functions_factory` better?